api/comments/serializers.py

```diff
@@ -0,0 +1,146 @@
+from rest_framework import serializers as ser
+from framework.guid.model import Guid
+from framework.auth.core import Auth
+from website.project.model import Node, Comment
+from rest_framework.exceptions import ValidationError, PermissionDenied, NotFound
+from api.base.exceptions import InvalidModelValueError
+from api.base.utils import absolute_reverse
+from api.base.serializers import (JSONAPISerializer,
+                                  JSONAPIHyperlinkedRelatedField,
+                                  JSONAPIHyperlinkedGuidRelatedField,
+                                  JSONAPIHyperlinkedIdentityField,
+                                  IDField, TypeField, LinksField,
+                                  AuthorizedCharField)
+
+
+class CommentReport(object):
```

Comment:
This class is just for the purpose of serializing comment report data into a format that is more compatible with the API, since that information is in the `Comment` model's `reports` field and not it's own model.